docusaurus/docs-internals/editor-controls/data-controls/codeMirror.md

```diff
@@ -252,3 +334,126 @@ In the examples below, we specify the language in which we will write the code.
   }
 }
 ```
+
+#### Default value examples
+
+In this example, the `codeMirror` control, which has the default value `".brz-button { background-color: purple; }"`, will set the background to `purple`.
+```js
+{
+  id: "code", 
+  type: "codeMirror",
+  default: {
+    value: ".brz-button { background-color: purple; }" 
+  }
+}
+```
+
+
+#### CSS examples
+
+Alter the appearance of `.brz-text` elements dynamically using CSS based on the content of a `codeMirror` component (id: "code"). When the component's value is empty, the text color changes to `green` with `underlined` styling. Otherwise, the text color switches to `blue` with `italicized` formatting within the specified wrapper (`{{WRAPPER}}`).
+
+```js
+{
+  id: "code",
+  type: "codeMirror",
+  style: ({ value }) => { 
+   if (value.value === "") {
+    return {
+      "{{WRAPPER}} .brz-button": {
+        "color": "green", 
+        "text-decoration": "underline"
+      }
+    };
+  }
+     
+  return {
+    "{{WRAPPER}} .brz-button": {
+      "color": "blue", 
+      "font-style": "italic"
+    }
+   };
+  }
+}
+
+```
+
+Retrieve the CSS styles `normal` and `active` for the color of the `.brz-button` element. For example, if a value is filled in the `normal` state and removed in the `active` state, then `.brz-button` will have a `black` color, and `.brz-button.active` will have a `red` color.
+
+```js
+{
+  id: "code",
+    type: "codeMirror",
+    states: ["normal", "active"],
+    style: ({ value }) => {
+    return {
+      "{{WRAPPER}} .brz-button": {
+        color: value.value === "" ? "black" : "red"
+      }
+    }
+  }
+}
+```
+
+#### Usage in HTML example
+
+In the example below, a custom button component for the Brizy editor is defined that supports advanced customization through the injection of `customCSS`. By using the `dangerouslySetInnerHTML` property, it allows users to add and style their button with their own CSS, which can be configured via the Brizy editor interface. The component is registered with Brizy and is equipped with a toolbar option to facilitate these advanced settings.
+
+```tsx
+import { Brizy } from "@brizy/core";
+import { Icon } from "./Icon";
+import React, { JSX } from "react";
+
+interface Props {
+  customCSS: string
+}
+
+const Button = (props: Props): JSX.Element => {
+  const { custom } = props;
+  
+ 
+  return (
+    <div className="brz-button">
```

Comment:
in your example you render HTML inside some "button" component, better change the name as example to "container" to be more explicit